src/Domain/Model/Comment.php

```diff
@@ -72,6 +61,21 @@ public function attachFile(string $path)
         );
     }
 
+    public function sendNewCommentEmail(): void
```

Comment:
I'm not happy with that name. Notice, that the only thing is happening in this function is triggering an event, not even a word about email sending (and it should indeed not be there) :) Why not just `commendOrder`?

Reply:
What about `triggerNewCommentEmail` because the function does not comment the Order, but just release the event that causes the email being sent :)

Reply:
Or `triggerNewCommentNotification`

Reply:
But from the point of this function, you don't know what will be the reaction for a thrown event. It can be a notification, an email, but also running a dishwasher 😄 The name should only explain what is happening inside this specific function... and because it's a domain layer, it would be good to now consist of any technical details (that's why I'm not into `trigger...` name 🚀).
So maybe just, as an event name, `orderCommented`? Need more ideas, cc @bartoszpietrzak1994 @pamil. Naming is hard 😄

Reply:
The two hardest things in the universe are caching and naming things. :D I like `orderCommented` though.

Reply:
Its gonna be `orderCommented` now :P